packages/spright-components/package.json

```diff
@@ -49,7 +49,7 @@
   "homepage": "https://github.com/ni/nimble#readme",
   "dependencies": {
     "@ni/fast-element": "^10.0.0",
-    "@ni/fast-foundation": "^10.0.0",
+    "@ni/fast-foundation": "^10.1.0",
```

Comment:
Can you search for and bump the fast-react-wrapper version as well? https://github.com/ni/nimble/pull/2646#issue-3193445400

Reply:
Oh, sure.